pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -5425,11 +5427,11 @@ CGROUP_SKB_HANDLE_FUNCTION(proto_tcp);
 CGROUP_SKB_HANDLE_FUNCTION(proto_tcp_dns);
 CGROUP_SKB_HANDLE_FUNCTION(proto_tcp_http);
 CGROUP_SKB_HANDLE_FUNCTION(proto_tcp_socks5);
+CGROUP_SKB_HANDLE_FUNCTION(proto_tcp_ssh);
 CGROUP_SKB_HANDLE_FUNCTION(proto_udp);
 CGROUP_SKB_HANDLE_FUNCTION(proto_udp_dns);
 CGROUP_SKB_HANDLE_FUNCTION(proto_icmp);
 CGROUP_SKB_HANDLE_FUNCTION(proto_icmpv6);
-CGROUP_SKB_HANDLE_FUNCTION(proto_socks5);
```

Comment:
[Need Info] is it intended to remove this SKB handle function?

Reply:
Haha, we only ever declared the function prototype, but never the body. This is fine, as we do not need this function, hence I cleaned it up.